app/build.gradle

```diff
@@ -47,6 +47,7 @@ dependencies {
     implementation 'com.google.firebase:firebase-analytics'
     implementation 'com.android.support:multidex:1.0.3'
     implementation 'com.google.firebase:firebase-firestore:23.0.4'
+    implementation "io.grpc:grpc-okhttp:1.32.2"
```

Comment:
What is the reason for this dependency?

Reply:
For some reason if I don't include this dependency the app crashes on my machine when I attempt to open a Firestore database. This is related to the "Method getAlpnSelectedProtocol not supported for object SSL socket over Socket" error mentioned eariler

Reply:
Okay no problem, thanks for the explanation.